Interface.C

```diff
@@ -102,16 +102,20 @@ void preciceAdapter::Interface::configureMesh(const fvMesh& mesh)
             {
                 const pointField faceNodes =
                     mesh.boundaryMesh()[patchIDs_.at(j)].localPoints();
+                const auto faceNodesSize = faceNodes.size();
                 //Allocate memory for z-coordinates
                 std::array<double, 2> z_location({0, 0});
                 constexpr unsigned int z_axis = 2;
 
                 // Find out about the existing planes
                 // Store z-coordinate of the first layer
-                z_location[0] = faceNodes[0][z_axis];
+                if (faceNodesSize > 0)
+                {
+                    z_location[0] = faceNodes[0][z_axis];
```

Comment:
This is fine now, but we could maybe have detected this situation a bit more easily if we had here `faceNodes.at(0)[z_axis]` (it would throw an exception instead of exploding).

Reply:
True, but won't change it here for style compatibility reasons.